style.css

```diff
@@ -327,6 +332,125 @@ button:hover {
   margin-top: 2rem;
 }
 
+/* ---Popup Window-- */
+
+.popup {
+  visibility: visible;
+  position: fixed;
+  z-index: 60;
+  height: 813px;
+  padding-top: 0.5rem;
+  width: 88%;
+  left: 4.5%;
+  top: 7.6rem;
+  border-radius: 16px;
+  margin-top: -7rem;
+  align-items: center;
+  background-color: white;
+}
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up: if you can see that the desktop pop-up doesn't look like the Figma design when on the screen size of `1450` px and above, the language list and the button tends to flow onto the Image. Check below :point_down:  the comparison between your design and Figma design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-08-02 16-36-03](https://github.com/denigit5/Portfolio/assets/45226848/d8abdac0-505e-4e68-ad9b-398fda448081)

**Figma Design**
![Screenshot from 2023-08-02 17-06-29](https://github.com/denigit5/Portfolio/assets/45226848/00d1399d-fe2a-461a-8fff-da8bfcf78111)